MatrixSDK/Utils/MXEventAssetTypeMapper.swift

```diff
@@ -0,0 +1,54 @@
+// 
+// Copyright 2022 The Matrix.org Foundation C.I.C
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+// http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+//
+
+import Foundation
+
+/// MXEventAssetTypeMapper enables to get the corresponding MXEventAssetType from event key String and the other way around.
+@objcMembers
+public class MXEventAssetTypeMapper: NSObject {
+    
+    // MARK: - Public
+    
+    /// Return event key String associate to the MXEventAssetType given
+    public func getEventKey(from eventAssetType: MXEventAssetType) -> String {
```

Comment:
As this is simply mapping one value to another, it probably doesn't need the `get` :)
```suggestion
    public func eventKey(for eventAssetType: MXEventAssetType) -> String {
```